cli/cmd/tap.go

```diff
@@ -259,19 +259,19 @@ func renderTapEvent(event *common.TapEvent) string {
 		dst,
 	)
 
-	http := event.GetHttp()
-	httpEvent := http.Event
+	httpEvent := event.GetHttp().GetEvent()
 	switch ev := httpEvent.(type) {
 	case *common.TapEvent_Http_RequestInit_:
 		return fmt.Sprintf("req id=%d:%d %s :method=%s :authority=%s :path=%s secured=%s",
 			ev.RequestInit.Id.Base,
 			ev.RequestInit.Id.Stream,
```

Comment:
Maybe we should switch these as well?

```
			ev.RequestInit.GetId().GetBase(),
			ev.RequestInit.GetId().GetStream(),
```

Or is it safe to assume that `Id` is always present? If you do decide to update it, can you update it in each of the following cases as well?